book/website/reproducible-research/open/open-access.md

```diff
@@ -4,32 +4,73 @@
+A quote from the Budapest Open Access Initative sums up the objective:
+>"We mean its free availability on the public internet, permitting any users to read, download, copy, distribute, print, search, or link to the full texts of these articles...the only constraint on reproduction and distribution, and the only role for copyright in this domain, should be to give authors control over the integrity of their work and the right to be properly acknowledged and cited"
+
+It has been estimated around 28% of the scholarly literature is Open Access {cite}`Piwowar2018stateofOA`and in 2020 more outputs were published through Open Access channels than traditional subscription channels globally {cite}`Dimensions2021openaccesssurpasses`.
+
+(rr-open-access-routes)=
+## Routes for Open Access
+(rr-open-access-preprints)=
+### Preprints
+
+A preprint is a scientific manuscript that is uploaded by the authors to a public server before peer review. 
+The first preprint server called arXiv was established in 1991 for physical sciences. 
+Likely due to the inefficiency of traditonal journals, preprint servers for other disciplines have grown hugely in popularity over the last 5 years, with the emergence of bioRxiv, medRxiv, ChemRxiv, SocArXiv and many more.
+Preprints allow researchers to get their results out quickly and gives the opportunity to get feedback on a manuscript before submitting it to a journal.
+Some journals will not allow a paper to be submitted if its already been posted on a preprint server as they consider it 'prior publication', however the ongoing popularity of preprints has forced many publishers to embrace preprints, or in the case of journals like eLife, actively encourage them {cite}`elife2020preprints`.
+You can find out more about preprints at [ASAPbio](https://asapbio.org/preprint-info).
+
+(rr-open-access-journal-publishing)=
+### Open Access Journal publishing
+
+Traditionally many journals are subscription journals, meaning their content is behind a paywall. 
+Universities may have subscriptions to that journal, allowing you to access its content if you are based at a subscribing institution or you may be able to pay a one off fee to access an individual article.
+
+The Open Access movement led to the creation of Open Access journals, where the content is open for anyone to access. 
+The number of these journals is still increasing rapidly, with 17,000 listed in the Directory of Open Access Journals [DOAJ](http://www.doaj.org) (data fetched November 2021). 
+
+(rr-open-access-gold-publishing)=
+#### Gold Open Access publishing
+
+Gold Open Access Publishing (or sometimes referred to as libre) can be defined as paying an Article Processing Charge (APC) to a journal so they publish the final version of your article, which is then permanently and freely available online for anyone.
```

Comment:
"publish the final version of your article **under an open access license**, which is then...."

Reply:
done :)